src/lib.rs

```diff
@@ -32,7 +32,7 @@ pub struct FormatOptions {
     /// When set, changes reserved keywords to ALL CAPS
     ///
     /// Default: false
```

Comment:
We should update the docblock here to specify the new behavior.

```suggestion
    /// When true, changes reserved keywords to ALL CAPS.
    /// When false, changes reserved keywords to all lowercase.
    ///
    /// Default: None
```